test/Directory.Build.props

```diff
@@ -0,0 +1,16 @@
+<Project>
+
+  <PropertyGroup>
+    <IsTestProject Condition="$(MSBuildProjectName.EndsWith('Tests'))">true</IsTestProject>
```

Comment:
I don't like this rule since it requires an exception (see comment below). I don't think this needs to be automatic, we can just set IsTestProject explicitly in all the projects that are test projects. I think it would be more readable